docs/requirements/requirements.md

```diff
@@ -0,0 +1,73 @@
+|Webserver|Tomcat|Should have| |
+|Platform| Windows, OSX | Should have | |
+|Language|Java|Should Have|No node.js (Must have)|
+
+
+### User Interface (UI)
+| What | How | Importance |Comment|
+|------|-----|------------|-------|
+|Ease of Use | Very Simple | Must Have | The UI must be simple and easy to use. Within a handful of clicks the transformation should be achieved |
+|Look and Feel  | The UI must look awesome. No further requirements. | Must Have | |
+|Type|Web Application|Must have| |
+|Supported browsers|Firefox, Chrome| Must have| |
+|Communcation|REST-calls|Must have| |
+|Technology|Angular|Should have| |
+
+### Command Line Interface (CLI)
+| What | How | Importance |Comment|
+|------|-----|------------|-------|
+|Communication|REST-calls|Must have| |
+
+### Transformations
+| What | How | Importance |Comment|
+|------|-----|------------|-------|
+|Target Host OS| Linux | Must have | |
+| Validation of input (CSAR) | syntax | Must Have | | 
+| Validation of input (CSAR) | semantics | Must Have | Not before execution; but if an error is found while executing the transformation, an error must be reported | 
+|TOSCA --> Cloud Foundry| CSAR -> Cloud Foundry Artifacts & Orchestration, if needed|Should have| it is acceptable to optimize the topology |
+|Support at least 3 target platforms | | Should have |  Transformation to every supported target platform must function properly |
+| TOSCA Spec version | Latest YAML specification | Should have | if YAML seems not to be working out, switching to the older XML specification is acceptable |
+|TOSCA --> AWS Cloud Formation| CSAR -> Cloud Formation Template & Orchestration, if needed|Should have| Preferable as first transformation; it is acceptable to optimize the topology |
+|TOSCA --> Kubernetes |  CSAR -> Kubernetes Artifacts & Orchestration, if needed|Should have| it is acceptable to optimize the topology|
+| Topology Optimization Mode | Implement normal and topology optimizing transformation mode | Could have| 'Tuning option' |
+| Adapt to target platform updates | If target platforms change, change transformator accordingly | Won't have | Beyound the scope of project; and most probably not relevant due to backward compatibilty | 
+
+### TOSCA Support 
+| What | How | Importance |Comment|
+|------|-----|------------|-------|
+| Specification support | most important elements | Must Have | Not included: Policies. Leave out the bells and whistles ('fancy stuff'). However, build architecture in an extendable way, so the full TOSCA spec can be supported in the future |
+| Node Support | Must be deployable  via Lifecycle Interface | Must have | |
+| NodeType, RelationshipType Support | Support from YAML-Profile | Must have | A basic LAMP-Stack must be deployable with the transformator; the level of mininum node type and relationship type support can be derived from this requirement |
+| NodeType Support | Support custom Node Types of Institute | Could Have | |
```

Comment:
Support custom Node Types of Institute -> Support custom **NodeTypes** of Institute
(uniformity)